docs/Custom-Protos.md

```diff
@@ -0,0 +1,167 @@
+# Creating custom protobuffer messages
+
+Unity and Python communicate by sending protobuffer messages to and from each other. You can create custom protobuffer messages if you want to exchange structured data beyond what is included by default. 
+
+Whenever you change the fields of a custom message, you must run `protobuf-definitions/make.bat` to create C# and Python files corresponding to the new message. Follow the directions in that file for guidance. After running it, reinstall the Python package by running `pip install ml-agents` and make sure your Unity project is using the newly-generated version of `UnitySDK`.
```

Comment:
Won't `pip install ml-agents` install the pypi version, not the version with the modified protobuf definitions?

Reply:
I've tried to clarify that you are meant to give the path to the local `ml-agents` folder, since if `pip` sees it's been given a valid local path it will try to install that before searching pypi.